Plugins/Sycl/CMakeLists.txt

```diff
@@ -0,0 +1,44 @@
+# This module needs at least CMake 3.13.
+cmake_minimum_required( VERSION 3.13 )
+
+find_package(SYCL REQUIRED)
+
+add_library(
+  ActsPluginSycl SHARED
+  # header files
+  include/Acts/Plugins/Sycl/Seeding/detail/Types.h
+  include/Acts/Plugins/Sycl/Seeding/CreateSeedsForGroupSycl.h
+  include/Acts/Plugins/Sycl/Seeding/DeviceExperimentCuts.hpp
+  include/Acts/Plugins/Sycl/Seeding/Seedfinder.hpp
+  include/Acts/Plugins/Sycl/Seeding/Seedfinder.ipp
+  include/Acts/Plugins/Sycl/Utilities/CreateQueue.h
+  include/Acts/Plugins/Sycl/Utilities/DeviceSelector.h
+  include/Acts/Plugins/Sycl/Utilities/ListPlatforms.h
```

Comment:
As I had to realise, there are a bunch of headers with a `.h` suffix, not just the one that I noticed before. These should all become `.hpp`. (I guess `.h` would be reasonable if it was implemented in a pure C `.c` file, but that's not what we do here.)

Also, don't forget to update the names here, in the CMake configuration. `CreateSeedsForGroupSycl.h` is already renamed to `.hpp`, but the CMake code still has its old name. (Leading to a cmake configuration error.)